management/src/test/resources/clusteredConfiguration.txt

```diff
@@ -0,0 +1,24 @@
+caches:
+    cache-1:
+        keyType: java.lang.String
+        valueType: java.lang.String
+        serviceConfigurations:
+        evictionAdvisor:
```

Comment:
purely from a human reading perspective, I find those nulls to just add extra noise. But maybe you have a good reason to include them?